### PR TITLE
Support full words (like infrastructure) instead of just keys like infra

### LINKED ISSUE #30: Support full words (like infrastructure) instead of just keys like infra
**Describe the solution you'd like**
It would be great to support full words (like infrastructure, documentation etc) when requesting adds

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by sinchang**
closes #30

### CODE REVIEW

File: src/utils/parse-comment/index.js [hunk 1]
```diff
@@ -26,12 +28,10 @@ const validContributionTypes = [
     'video',
 ]
 
-// TODO
-// const contributionTypeMappings = {
-//     'event organizing': 'eventOrganizing',
```
Comment (jakebolam) [inline src/utils/parse-comment/index.js]: Do you think we could add these in too?

**Comment by jakebolam** [comment]
Thank you @sinchang, this is awesome! Left 1 comment

File: src/utils/parse-comment/index.js (hunk 1 above)
Comment (sinchang) [reply src/utils/parse-comment/index.js]: ok

File: test/utils/parse-comment/index.test.js [hunk 2]
```diff
@@ -42,6 +42,19 @@ describe('parseComment', () => {
             action: false,
         })
     })
+
+    test('Support full words (like infrastructure) instead of just keys like infra', () => {
```
[the remark this answers is not in the document]
Comment (sinchang) [reply test/utils/parse-comment/index.test.js]: sorry, have some problems when want to compile `user testing` to `userTesting `, any suggestions?

[the remark this answers is not in the document]
Comment (jakebolam) [reply test/utils/parse-comment/index.test.js]: Yep sure things, lets try the test and see what happens: https://github.com/all-contributors/all-contributors-bot/pull/34/commits/ca3e09aa41069b61051cf8f9de2b6a26d55a4500

File: test/utils/parse-comment/index.test.js [hunk 3]
```diff
@@ -42,6 +42,31 @@ describe('parseComment', () => {
...
+                `@${testBotName} please add jakebolam for infrastructure, documentation`,
+            ),
+        ).toEqual({
+            action: 'add',
+            who: 'jakebolam',
+            contributions: ['infra', 'doc'],
+        })
+    })
+
+    test('Support split words (like user testing)', () => {
```
Comment (jakebolam) [inline test/utils/parse-comment/index.test.js]: @sinchang looks like its not working yet, lets address this in another issue

**Comment by sinchang** [comment]
ci passed 🎉

**Comment by jakebolam** [comment]
Thanks so much.

@allcontributors please add sinchang for test and code

**Comment by jakebolam** [comment]
@all-contributors please add sinchang for test and code

**Pull request merged**